### PR TITLE
Deprecate Plugins and Packages that have been moved to `backstage/community-plugins`

### PR DESCRIPTION
**Pull request by benjdlambert**
https://github.com/backstage/community-plugins/pull/101 moves a lot of plugins into individual workspaces in the community-plugins repository.

This deprecates and marks them as moved for easy upgrading. Just imports might need to be updated in Backstage installations.

### CODE REVIEW

File: .changeset/migrate-1713465834005.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-adr': patch
+'@backstage/plugin-adr-backend': patch
+'@backstage/plugin-adr-common': patch
```
Comment (freben) [inline .changeset/migrate-1713465834005.md:4]: These three are repeated from the changeset above. I'm on mobile now but maybe worth double checking through these

File: .changeset/migrate-1713466136342.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-newrelic-dashboard': patch
```
Comment (freben) [inline .changeset/migrate-1713466136342.md:2]: One more

**Review by freben — COMMENTED** [review]
Ok i think there was just one more such occurrence

File: .changeset/migrate-1713465921873.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-circleci': patch
```
Comment (adamdmharvey) [inline .changeset/migrate-1713465921873.md:2]: For this one, should likely be a deprecate as the vendor moved this code to their own org in 2023:

https://github.com/CircleCI-Public/backstage-plugin

So this is still there but likely doesn't need to be? Could either tackle in this PR or maybe leave this one out ?

Cc @ribiero

**Review by adamdmharvey — COMMENTED** [review]
Nice Ben 👍 

Comment about CircleCI plugin added.

Also some of these are in CODEOWNERS, so should likely clean that up also ?

**Pull request merged**